### PR TITLE
[EasyApiToken] Fix broken test and add compatibility with auth0-PHP >= 8.3

### PR DESCRIPTION
**Pull request by alexndlm**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

File: packages/EasyApiToken/tests/AbstractAuth0JwtTokenTestCase.php [hunk 1]
```diff
@@ -45,16 +45,14 @@ protected function createAuth0JwtDriver(
         ?string $audienceForEncode = null,
         ?array $allowedAlgos = null
     ): JwtDriverInterface {
-        $auth0Driver = new Auth0JwtDriver(
+        return new Auth0JwtDriver(
             $validAudiences ?? static::$validAudiences,
             $authorizedIss ?? static::$authorizedIss,
-            'localhost',
+            'example.com',
```
Comment (alexndlm) [inline packages/EasyApiToken/tests/AbstractAuth0JwtTokenTestCase.php:51]: We need this after this excellent commit https://github.com/auth0/auth0-PHP/commit/e89127a97ad72715e1a55c57fc3455146e12d9a8

File: packages/EasyApiToken/src/External/Auth0JwtDriver.php [hunk 2]
```diff
@@ -106,6 +106,8 @@ public function decode(string $token): array
         $config = new Auth0V8SdkConfiguration([
             'domain' => $this->domain,
             'clientId' => 'client_id',
+            'strategy' => Auth0V8SdkConfiguration::STRATEGY_API,
```
Comment (alexndlm) [inline packages/EasyApiToken/src/External/Auth0JwtDriver.php:109]: We need this after this commit https://github.com/auth0/auth0-PHP/commit/a1fd98f8569649717cc50e18fb92521091c1312e

Comment (natepage) [reply packages/EasyApiToken/src/External/Auth0JwtDriver.php:109]: @alexndlm I think that would break applications that update the monorepo but not auth0

Comment (itorgov) [reply packages/EasyApiToken/src/External/Auth0JwtDriver.php:109]: Agree, we have to ensure that everything will work with version 8.0.

**Review by itorgov — CHANGES_REQUESTED** [review]

File: packages/EasyApiToken/src/External/Auth0JwtDriver.php (hunk 2 above)
Comment (alexndlm) [reply packages/EasyApiToken/src/External/Auth0JwtDriver.php:109]: @natepage these settings already exists in 8.0. From 8.3 these are required. Current changes should not impact apps.

**Review by itorgov — APPROVED** [review]

**Review by akl-95 — APPROVED** [review]

**Review by natepage — APPROVED** [review]

**Pull request merged**